docs/setup.md

```diff
@@ -106,6 +115,8 @@ $ kubectl get po -n kube-system -l app=local-volume-provisioner
 $ kubectl get pv | grep local-storage
 ```
 
+> Note: The local-volume plugin expects paths to be stable, if you remove a previous mount-point in the discovery directory (default to `/mnt/disks/`), you should remove the PV manually to keep consistency.
```

Comment:
First we need to `umount` the `/mnt/disks/local-pv01`, and then delete the two directories.

```
$ umount /mnt/disks/local-pv01
$ rm -rf /mnt/disks/local-pv01 
$ rm -rf /data/local-pv01
```